xarray/core/dataset.py

```diff
@@ -7681,7 +7681,7 @@ def map_blocks(
         ``func`` can work on numpy arrays, it is recommended to use ``apply_ufunc``.
 
         If none of the variables in this object is backed by dask arrays, calling this function is
-        equivalent to calling ``func(obj, *args, **kwargs)``.
+        equivalent to calling ``func(obj, *args, kwargs)``.
```

Comment:
I think the previous version was correct.

Reply:
Okay I'll look into it right away. If you have any advice, please let me know.

Reply:
I think you have to start over and only apply the changes in the docstring which you mentioned in the PR description.
All the other changes are actually wrong, since the current way is correct.